api/app/weather_models/__init__.py

```diff
@@ -55,6 +57,54 @@ def interpolate_between_two_points(  # pylint: disable=invalid-name
     return function(xn)
 
 
+def interpolate_bearing(time_a: datetime, time_b: datetime, target_time: datetime,
+                        direction_a: float, direction_b: float):
+    """ Interpolate between two bearings, along the acute angle between the two bearings.
+
+    params:
+    :time_a: Time of the first bearing.
+    :time_b: Time of the 2nd bearing.
+    :target_Time: Time for which we desire a value.
+    :direction_a: Bearing (in degrees) at time_a.
+    :direction_b: Bearing (in degrees) at time b.
+    """
+    x_axis = (time_a.timestamp(), time_b.timestamp())
+    if abs(direction_a - direction_b) > 180:
```

Comment:
Is there a nicer way to deal with obtuse angles? Anyone?

Reply:
Can you explain what this condition is checking?

Reply:
You want me to just move the comment on line 74 up?

Reply:
I can't think of a nicer way. The logic of it makes sense to me

Reply:
@Kyubinhan I added an explanation - check if it makes sense to you please.